core/runtime.rs

```diff
@@ -3434,19 +3508,21 @@ pub mod tests {
         _maybe_referrer: Option<ModuleSpecifier>,
         _is_dyn_import: bool,
       ) -> Pin<Box<ModuleSourceFuture>> {
+        eprintln!("load() should not be called");
         unreachable!()
       }
     }
```

Comment:
This test needs to be updated to load ~30-200 modules just to make sure that we can store that much data in the snapshot. It should be something auto-generated - like each module has a `getNumber()` function that returns an incrementing number - we'll then call all these functions from various modules and collect them to an array and then assert that the array has number from 1 through no of modules